src/augrun.c

```diff
@@ -1010,6 +1010,33 @@ static const struct command_def cmd_transform_def = {
     .help = cmd_transform_help
 };
 
+static void cmd_load_file(struct command *cmd) {
+    const char *file = arg_value(cmd, "file");
+    int r = 0;
+
+    r = aug_load_file(cmd->aug, file);
+    if (r == -1)
```

Comment:
It's more robust to check `r < 0` here (in case we ever use other negative numbers to indicate other error conditions)